static/index.html

```diff
@@ -202,20 +197,28 @@
                                 </div>
                                 <div class="card-header h6">Temperature</div>
                                 <div class="card-body">
-                                    <div class="row">
+                                    <div class="row g-3">
                                         <div class="col-6">
                                             <div class="input-group">
-                                                <div class="input-group-text mx-auto" id="bedGroup">
-                                                    <i class="icon icon-hotend" alt="hotend temperature"></i>
+                                                <div class="input-group-text mx-auto" id="nozzleGroup">
+                                                    <i class="icon icon-hotend"></i>
```

Comment:
I'm not against the idea, but in that case we need another way to do it.

Having `alt` on `<i>` literally makes the html invalid, since `<i>` doesn't have that attribute.